docs/node_running/node_providers.md

```diff
@@ -0,0 +1,43 @@
+# Node Providers
+
+To interact with Arbitrum One and the RinkArby testnet, you can rely on the same popular node providers that you are already using on Ethereum. There are multiple providers that may help you build on Arbitrum, here is a list of a couple of them.
+
+## [Alchemy](https://alchemy.com/?a=arbitrum-docs)
+
+Alchemy provides endpoints for both Arbitrum One (Mainnet) and Rinkeby testnet. To use these RPC endpoints, you need to create a free account and set up an ***Alchemy API key***  to authenticate your requests. For detailed steps of how to create one, see the [Alchemy's documentation](https://docs.alchemy.com/alchemy/introduction/getting-started#1.create-an-alchemy-key). 
+
+Next, depending one which network you want to interact with, you can choose between the two RPC endpoints provided by Alchemy and interact with them directly in the command line or point your [wallet's](https://portal.arbitrum.one/#wallets) RPC endpoints to them. For more information on Arbitrum JSON-RPC API methods supported by Alchemy, check out the [official documentation](https://docs.alchemy.com/alchemy/apis/arbitrum).
+
+Alchemy provides 300M compute units per month for free, or the equivalent of 30M getLatestBlock requests. Additional usage requires upgrading to a Growth Tier ($49 / month).
+
+## [Infura](https://infura.io/)
+
+Infura is a node service provider that provides access to Arbitrum One (Mainnet) and Rinkeby testnet. To start interacting with either of these networks, you need to set up an Infura account and create a project. Next, you will be given a ***Project ID*** (for detailed steps, check out the [Infura's documentation](https://blog.infura.io/getting-started-with-infura-28e41844cc89/)). You can now select which endpoint you want to connect to, this could be Arbitrum One, or it could be Rinkeby testnet. 
+
+Infura provides access to Arbitrum up to 100k requests per day for free. Once Arbitrum leaves beta in Infura, it will be charged as an addon of $200 / month.
```

Comment:
Should replace the two lines above as requested by Infura

Infura is a node service provider that provides access to Arbitrum One (Mainnet) and Rinkeby testnet. To start interacting with either of these networks, you need to set up an Infura account here<https://infura.io/register> and create a new key. For detailed steps, check out the Infura's documentation <Please replace current url with updated docs url https://docs.infura.io/infura/networks/arbitrum>
You can now select which endpoint you want to connect to, this could be Arbitrum One, or it could be Rinkeby testnet.
Infura provides access to Arbitrum across all their subscription plans (Core Free, Developer, Team & Growth). See pricing & daily request rate limits here <Url https://infura.io/pricing>

Reply:
updated